parsec/core/gui/custom_dialogs.py

```diff
@@ -91,8 +92,8 @@ def _bring_to_top(pid_target):
         callback_type = ctypes.WINFUNCTYPE(win.BOOL, win.HWND, win.LPARAM)
 
         # Callback enumerating the windows
-        def enum_windows_callback(hwnd, param):
-            nonlocal success
+        def enum_windows_callback(hwnd: win.HWND, param: object) -> bool:
+            nonlocal success  # type: ignore[misc]
```

Comment:
What are the reason of the ignore ?

Reply:
On windows mypy tells that this variable is unbound which is not the case at runtime. I'll add a comment.

Reply:
This isn't the real reason, the reason is that the `nonlocal` `success` is defined in `l.108`
If you move the definition of the function between the variable definition and the function call (l.110), the error should disappear